corehq/ex-submodules/casexml/apps/stock/models_v2.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class LedgerValue(models.Model):
+    """
+    Represents the current state of a ledger. Supercedes StockState
+    """
+    # domain not included and assumed to be accessed through the foreign key to the case table. legit?
```

Comment:
Same goes for location?  Looks like `CommCareCaseSQL` doesn't point to locations, but it seems like it should (via a nullable foreign key or 1-to-1)